### ISSUE TITLE
How to fine-tune ArcFace on a new dataset?

### ISSUE THREAD

**Issue by sunshineatnoon** [deepinsight/insightface#2089]
Hi, Thanks for this awesome work. I'm trying to use ArcFace on FFHQ. However, it seems there's domain gap so the running mean and variance does not directly fit to FFHQ. Has anyone tried to fine-tune ArcFace on other datasets and how to handle the BatchNorm layers?

I also tried to freeze the BatchNorm layers, which unfortunately leads to NaN.

**Comment by lamnt-nd994** [comment]
Have you solved this problem yet?
